app/assets/stylesheets/addPage.scss

```diff
@@ -38,40 +38,37 @@ h4 {
 	text-decoration: underline;
 }
 
-label{
+label {
 	text-align: center;
 }
 
-.element_div{
+.element_div {
 	display: flex;
 	flex-direction: column;
 	justify-content: space-around;
 	border-bottom: groove;
 	padding: 10px;
 }
 
-#submit_div{
+#submit_div {
 	display: flex;
 	justify-content: space-around;
 	flex-wrap: nowrap;
 	padding: 10px;
 }
 
-#submit_button:hover{
-	background-color: darken($blue, 10%)
+#submit_button:hover {
+	background-color: darken($blue, 10%);
 }
 
-#cancel_button:hover{
-    background-color: darken($red, 10%)
+#cancel_button:hover {
+    background-color: darken($red, 10%);
 }
 
-.checkboxClass{
-	display: grid;
-	grid-template-columns: 3fr 1fr;
+select {
+  widtH: 50%;
```

Comment:
The fact that Github marks this as red is problematic—I don’t immediately see why it’s red but Github is catching something we aren’t.

Reply:
I think this was a typo because the "h" was capitalized. I've fixed this typo.